frodo.js

```diff
@@ -9,6 +9,7 @@
  * Copyright 2016 Henrik Ingo (@henrikingo)
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?